plugin/src/main/resources/com/redhat/jenkins/plugins/ci/pipeline/CIMessageSenderStep/help-timeToLiveMillis.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    <p>Sets the length of time in minutes from its dispatch time that a produced message
```

Comment:
The filename indicates millisecond resolution.

```suggestion
    <p>Sets the length of time in milliseconds from its dispatch time that a produced message
```

Reply:
This was fixed. The file was renamed to `help-timeToLiveMinutes.html`.